helper-hardhat-config.js

```diff
@@ -30,14 +36,20 @@ const networkConfig = {
         oracle: '0x7AFe1118Ea78C1eae84ca8feE5C65Bc76CcF879e',
         jobId: '6d1bfe27e7034b1d87b5270556b17277',
         fee: '100000000000000000',
+        autoFund: '1',
+        fundAmount: "1000000000000000000"
     },
     1: {
         name: 'mainnet',
-        linkToken: '0x514910771af9ca656af840dff83e8264ecf986ca'
+        linkToken: '0x514910771af9ca656af840dff83e8264ecf986ca',
+        autoFund: '0',
+        fundAmount: "0"
     },
     5: {
         name: 'goerli',
-        linkToken: '0x326c977e6efc84e512bb9c30f76e30c160ed06fb'
+        linkToken: '0x326c977e6efc84e512bb9c30f76e30c160ed06fb',
+        autoFund: '0',
```

Comment:
How come this is `0` and the other testnets are `1000000000000000000`?

Reply:
Looking at the [docs](https://docs.chain.link/docs/vrf-contracts) for VRF & Any-API, don't think they are live on Goerli testnet. So there's no need to pre-fund by default